platforms/anchor-platform/admin-guide/sep24/configuration.mdx

```diff
@@ -109,6 +113,8 @@ SECRET_SEP24_MORE_INFO_URL_JWT_SECRET="your encryption key shared with your busi
 
 `SECRET_SEP24_INTERACTIVE_URL_JWT_SECRET` and `SECRET_SEP24_MORE_INFO_URL_JWT_SECRET` are encryption keys that the Anchor Platform will use to generate short-lived tokens it will add to the URLs provided to the wallet. Your business server must also have these keys in its environment so it can verify the token's signature.
 
+`SEP24_INITIAL_USER_DEADLINE_SECONDS` is an optional param that defines the time in seconds a user has to act before the transaction moves to the next status. It determines the `user_action_required_by` field, which indicates the deadline. Check [JSON-RPC Methods][json-rpc-methods] for usage examples.
```

Comment:
Do you mind adding a paragraph of how it's supposed to be used?
(i.e. business should call get /transaction and check with user_action_required_by has passed, and manually execute the logic. For example, they can expire the transaction, or auto-refund, depending on the tx status)

Reply:
I do want to keep this part focused on all the configuration and do not stretch too much.

The use of `user_action_required_by ` should go to where's really being applied, e.g RPC method section, or [platform transaction](https://developers.stellar.org/platforms/anchor-platform/api-reference/platform/transactions/get-transactions)

Reply:
Hmm that's fair. Maybe we should include it in SEP-24/SEP-6 guide instead? [This section](https://developers.stellar.org/platforms/anchor-platform/admin-guide/sep24/integration#expired-transaction) makes sense to include this information in.